Packs/CheckpointFirewall/Integrations/CheckPointFirewall_v2/CheckPointFirewall_v2.py

```diff
@@ -1454,14 +1467,15 @@ def checkpoint_list_package_command(client: Client, identifier: str) -> CommandR
     if result:
 
         gwinfo = result.get('installation-targets-revision')
-        for element in gwinfo:
-            current_printable_result = {}
-            current_printable_result['name'] = result.get('name')
-            for endpoint in headers:
-                current_printable_result[endpoint] = element.get(endpoint)
-            printable_result.append(current_printable_result)
-        readable_output = tableToMarkdown('CheckPoint data for package:', printable_result,
-                                          headers, removeNull=True)
+        if gwinfo:
+            for element in gwinfo:
+                current_printable_result = {}
+                current_printable_result['name'] = result.get('name')
+                for endpoint in headers:
+                    current_printable_result[endpoint] = element.get(endpoint)
+                printable_result.append(current_printable_result)
+            readable_output = tableToMarkdown('CheckPoint data for package:', printable_result,
```

Comment:
if `gwinfo` is empty (the same will happen if `result` is empty), the entry will have no text